internal/configuration/configuration.go

```diff
@@ -14,6 +14,7 @@ type Configuration struct {
 	Port                int32
 	PortPool            server.Pooler
 	TunnelServerAddress string
+	PrivateKey          string
```

Comment:
@mpetrun5 @mpetrunic I am not sure if this is the best place to hold the private key. I could embed it as the private property of `ApiController`. I need to save it somehow so I could verify signatures in stats handler.

Reply:
I refactored this